docs/UserGuide.md

```diff
@@ -106,15 +166,60 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [pr/PROPERTY]… [
 - At least one of the optional fields must be provided.
 - Existing values will be updated to the input values.
 - Type of the user can be changed from `buyer` to `seller` & vice versa by adding a `Property` or a `Preference` which removes the user's current `Property` or `Preference`.
```

Comment:
Same here

Reply:
Instances of `user` where `client` is more apporiate have been changed.